### PR TITLE
[MNG-7848] Change the notification recipients in s390x pipeline

### PR DESCRIPTION
**Pull request by kun-lu20**
This PR aims to change the notification recipients in `Jenkinsfile.s390x` to Linux on IBM Z Open Source Team.

-------------------------------------------------------------------------------------

Following this checklist to help us incorporate your
contribution quickly and easily:

 - [x] Make sure there is a [JIRA issue](https://issues.apache.org/jira/browse/MNG) filed
       for the change (usually before you start working on it).  Trivial changes like typos do not
       require a JIRA issue. Your pull request should address just this issue, without
       pulling in other changes.
 - [x] Each commit in the pull request should have a meaningful subject line and body.
 - [x] Format the pull request title like `[MNG-XXX] SUMMARY`,
       where you replace `MNG-XXX` and `SUMMARY` with the appropriate JIRA issue.
 - [x] Also format the first line of the commit message like `[MNG-XXX] SUMMARY`.
       Best practice is to use the JIRA issue title in both the pull request title and in the first line of the commit message.
 - [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
 - [x] Run `mvn clean verify` to make sure basic checks pass. A more thorough check will
       be performed on your pull request automatically.
 - [x] You have run the [Core IT][core-its] successfully.

If your pull request is about ~20 lines of code you don't need to sign an
[Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf) if you are unsure
please ask on the developers list.

To make clear that you license your contribution under
the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0)
you have to acknowledge this by using the following check-box.

 - [x] I hereby declare this contribution to be licenced under the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0)

 - [x] In any other case, please file an [Apache Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

[core-its]: https://maven.apache.org/core-its/core-it-suite/

### CODE REVIEW

**Comment by kun-lu20** [comment]
Hi @olamy @slachiewicz ,

Could you please review this PR when you have a chance?

Thank you very much!

File: Jenkinsfile.s390x [hunk 1]
```diff
@@ -171,3 +171,74 @@ def archiveDirs(stageId, archives) {
...
+    for (def author in authors) {
+		if (author.matches('(.*)github(.*)')) sendMail = false
+    }
+    
+    if (authors.isEmpty()) sendMail = false
+    
+    if (sendMail) {
+        messageBody = messageBody + '\n${FAILED_TESTS}\n' + messageTail
+        println("Sending email ...")
+        emailext body: messageBody, replyTo: 'dev@maven.apache.org', subject: messageSubject, to: 'rishi@ca.ibm.com'
```
Comment (olamy) [inline Jenkinsfile.s390x:242]: can you change the `replyTo` ?

**Review by olamy — CHANGES_REQUESTED** [review]

File: Jenkinsfile.s390x (hunk 1 above)
Comment (kun-lu20) [reply Jenkinsfile.s390x]: @olamy Sure, I've changed it as well. Please take a look when you have some time.

**Comment by olamy** [comment]
> Hi @olamy,
> 
> Thank you very much! Could you please set up and run a nightly build s390x pipeline on Jenkins CI?

Hi
Sorry but I finally realised there is something else to change. See my comment here: https://github.com/apache/maven/pull/1207/files#r1285167844

**Comment by kun-lu20** [comment]
Hi @olamy ,

Thanks for pointing out! I'll raise a new PR to address it.

**Comment by jira-importer** [comment]
Resolve #8740

**Pull request merged**